op-service/txintent/burn_call.go

```diff
@@ -0,0 +1,32 @@
+package txintent
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/lmittmann/w3"
+)
+
+type GasBurnCall struct {
+	GasGuzzlerAddress common.Address
```

Comment:
This is descriptive enough, but nit: I don't really like the word "Guzzler" here and would prefer we use a more generic term:
- Burner
- Waster
- User